src/adapter.cpp

```diff
@@ -26,6 +26,7 @@ namespace D3D11On12
             Args.bSupportDisplayableTextures : false)
         , m_bSupportDeferredContexts(Args.D3D11On12InterfaceVersion >= 5 ?
             Args.bSupportDeferredContexts : true)
+        , m_bSupportsNewPresentPath(Args.Callbacks.Present11On12CB != nullptr)

```

Comment:
If D3D11.dll is old, the size of the struct it provides here will be smaller and this callback might be garbage instead of null. You need a `D3D11On12InterfaceVersion` check before you can access the new callback field.

Reply:
Ah true, thanks :)